octodns_dnsmadeeasy/__init__.py

```diff
@@ -102,10 +102,15 @@ def domain(self, name):
         return self._request('GET', path).json()
 
     def domain_create(self, name):
-        self._request('POST', '/', data={'name': name})
+        resp = self._request('POST', '/', data={'name': name})
+        _id = resp.json()['data'][0]['id']
```

Comment:
It's just going to be `resp.json()['id']`, the response doesn't have a list of data like the `domains` resp has

Reply:
Ah. I was probably getting a mock that was intended for a domains call.